test/serializer/abstract/Property2.js

```diff
@@ -13,12 +13,12 @@ if (!x) {
 
 let ob2 = { y: 2 };
 if (!x) {
-  Object.defineProperty(ob2, "x",
+  Object.defineProperty(ob2, "x2",
```

Comment:
This change is used to make two objects using different property names so that it's much easier to debug.